mail/gophish/patches/gophish-0.12.1-env-version.patch

```diff
@@ -0,0 +1,23 @@
+--- a/gophish.go
```

Comment:
Is this backport or not? If it is backport, then it should have Git header.
If it is not backported, then please add Git header, which contains commit header, description, who is the author, etc and most likely try to send it to the upstream first.

Reply:
Updated. I had already submitted this as https://github.com/gophish/gophish/pull/3062. Thank you.

Reply:
Could you update it once again from this link:
https://patch-diff.githubusercontent.com/raw/gophish/gophish/pull/3062.patch ?